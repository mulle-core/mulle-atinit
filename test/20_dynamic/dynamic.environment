MULLE_ATINIT_DEBUG=YES
