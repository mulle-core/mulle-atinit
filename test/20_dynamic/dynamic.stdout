x: "first"
z: "mid"
y: "last"
main
